Src/Tools/DiscoveryDocPatcher/Program.cs

```diff
@@ -36,7 +36,12 @@ static int Main(string[] args)
                 // I've let a comment on the script as well.
                 // PatchDirectory(discoveryDocPath);
 
-                PatchDataLabeling(discoveryDocPath);
+                // No need for this patch after moving to the microgenerator.
```

Comment:
Possibly just remove entirely, including the PatchDataLabeling method itself?

Reply:
Yep, was following what we do for the other patches, but it's true that this one won't happen again. On it.

Reply:
Done, PTAL.